### PR TITLE
feat: add Angular v15 support

### PR DESCRIPTION
**Pull request by aitboudad**
fix #3510

### CODE REVIEW

**Comment by cr7boulos** [comment]
@aitboudad Our team has really enjoyed using the Angular Formly library for our project work. We are currently in the process of migrating to Angular 15 and will most likely need support for the material-legacy UI theme until we migrate fully. What is the timeframe for this feature being released in a public version of Formly on npm?

**Comment by aitboudad** [comment]
@cr7boulos I think I'll be able to push a new release next week, hope I'll find some time for it :).

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.1.0](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.1.0) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by cr7boulos** [comment]
@aitboudad Thank you very much!

**Comment by dc-p8** [comment]
thanks you for the big work

There is a small bug with matSuffix using material mdc components : since the default form-wrapper is using the matSuffix directive, the default behaviour is to expect an icon (matSuffix is equivalent to matIconSuffix). The problem is that using matSuffix with text makes the text not aligned with the input.

**Comment by aitboudad** [comment]
@dc-p8 please create a new issue, we'll try to provide a fix whenever possible.

**Comment by aitboudad** [comment]
@dc-p8 see https://github.com/ngx-formly/ngx-formly/issues/3556#issuecomment-1399024567

**Comment by dc-p8** [comment]
Just tested and it's working great, text suffix is now correctly aligned with the input, thanks

File: src/ui/material/form-field/src/form-field.wrapper.ts [hunk 1]
```diff
@@ -58,13 +58,13 @@ export interface FormlyFieldProps extends CoreFormlyFieldProps {
         >
       </mat-label>
 
-      <ng-container matPrefix *ngIf="props.prefix">
+      <ng-containe matPrefix *ngIf="props.prefix">
```
Comment (Lehoczky) [inline src/ui/material/form-field/src/form-field.wrapper.ts:61]: @aitboudad Was this renaming from `ng-container` to `ng-containe` intentional?

Comment (aitboudad) [reply src/ui/material/form-field/src/form-field.wrapper.ts:61]: @Lehoczky not intentional, could you please send a PR. Surprisingly it works :thinking:!

Comment (Lehoczky) [reply src/ui/material/form-field/src/form-field.wrapper.ts:61]: Sure thing!

**Pull request merged**